components/proxy/src/main/java/com/hotels/styx/StyxServer.java

```diff
@@ -97,8 +97,10 @@ private static StyxServer createStyxServer(String[] args) {
         LOG.info("Styx configFileLocation={}", startupConfig.configFileLocation());
         LOG.info("Styx logConfigLocation={}", startupConfig.logConfigLocation());
 
+        StyxConfig styxConfig = parseConfiguration(startupConfig);
+
         StyxServerComponents components = new StyxServerComponents.Builder()
-                .styxConfig(parseConfiguration(startupConfig))
+                .styxConfig(styxConfig)
                 .startupConfig(startupConfig)
```

Comment:
Is this just a cosmetic change? Please consider backing this off to keep the change history clean.

Reply:
reverted